### PR TITLE
Overview topics for styling, components, patterns

### PR DESCRIPTION
**Pull request by StefanIvanov**
Updating overview for styling
Adding overview for components and patterns

### CODE REVIEW

File: en/components/components-overview.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Components Overview - Design System Components
+_description: The Indigo Design System contains 60+ UI Components with numerous presets, states and elaborate styling capabilities built in.
```
Comment (simeonoff) [inline en/components/components-overview.md:3]:
```suggestion
_description: The Indigo Design System contains 60+ UI Components with numerous presets, states, and elaborate built-in styling capabilities.
```

File: en/components/components-overview.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+# Components Overview
+
+The **Indigo.Design System** empowers your design workflow with 60+ UI Components that also map to Ignite UI for Angular controls. Each component is optimized for responsive web development and provides various templating capabilities through smart layout and resizing rules set in the library. Furhtermore, we have added light/dark modes and various interaction states to choose from when designing dynamic experiences. All this streamlines application design and development allowing any user interface created with the library in Sketch to be built in Angular with Ignite UI. 
+
+The Sketch library has a `🧩 Components` page containing all Components and symbols related to them. Below is a small part of it showing some of them such as the different types of [Buttons](button.md), [Button Group](button-group.md), and form elemenst like [Checkbox](checkbox.md), [Switch](switch.md) and [Radio Group](radio-group.md).
```
Comment (simeonoff) [inline en/components/components-overview.md:11]:
```suggestion
The Sketch library has a `🧩 Components` page containing all Components and symbols related to them. Below is a small part of it showing some of them such as the different types of [Buttons](button.md), [Button Group](button-group.md) and form elemenst like [Checkbox](checkbox.md), [Switch](switch.md), and [Radio Group](radio-group.md).
```

File: en/components/components-overview.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+<img class="responsive-img" src="../images/components-page.png" />
+
+All **Indigo.Design System** Components are built on top of rigid [styling foundations](../styling/styling-overview.md) letting you leverage an elaborate mechanism for theming and branding your user interface.
+
+> [!WARNING]
+> Unlinking any component instance from the library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability for the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
```
Comment (simeonoff) [inline en/components/components-overview.md:18]:
```suggestion
> Unlinking any component instance from the library makes it impossible to automatically apply updates from subsequent versions of the Design System and may impact the code generation capability of the component. The code generation services rely on specially designated `🚫 metadata` layers, containing metadata descriptions of the state, template and other properties of the component.
```

File: en/patterns/patterns-overview.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- 🌆 Scheduling
+- 🌆 Social Feed
+- 🌆 Tiles
+- 🌆 User Profile
+
+Unlike a Component, every Pattern can be detached to show as a group, preserving the look and feel you have defined before that. Below is a snippet of some of the [list patterns](lists.md) such as a contacts list, searchable and filterable lists, and list empty states when no matching results are found.
+
+<img class="responsive-img" src="../images/patterns_overview.png" />
+
+We constantly improve our patterns and add new ones. To identify changes more easily from one release of the **Indigo.Design System** to the next, you will notice that we append a 🆕 emoji to the pages containing updates. You may also create your own Patterns from interface layouts that you use over and over again in your designs, extending our default set and giving your design productivity a further boost.
```
Comment (simeonoff) [inline en/patterns/patterns-overview.md:36]:
```suggestion
We constantly improve our patterns by adding new ones. To identify changes more easily from one release of the **Indigo.Design System** to the next, you will notice that we append a 🆕 emoji to the pages containing updates. You may also create your own Patterns from interface layouts that you use over and over again in your designs, extending our default set and giving your design productivity a further boost.
```

File: en/style/styling-overview.md [hunk 5]
```diff
@@ -1,31 +1,36 @@
...
-_description: The Styling Library contains everything for you to be able to define themes for both Components and Patterns.
+_description: The Indigo.Design System contains everything for you to be able to define themes for both Components and Patterns.
 _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
 ---
 
-# Overview
+# Styling Overview
 
-All Components map to Ignite UI for Angular components. The theming engine of Infragistics' Angular product, through powerful sass functions and mixins, allows for the creation of custom color themes that can restyle the whole product in just a few lines of code.
+The **Indigo.Design System** provides elaborate means for styling all 60+ UI components that also map to Ignite UI for Angular controls. The theming engine of Infragistics' Angular product, through powerful sass functions and mixins, allows for the creation of custom color themes that can restyle the whole product in just a few lines of code in a similar fashion.
```
Comment (simeonoff) [inline en/style/styling-overview.md:9]:
```suggestion
The **Indigo.Design System** provides elaborate means for styling all 60+ UI components that also map to Ignite UI for Angular controls. The theming engine in our Angular product allows for the creation of custom color themes that can restyle the whole product in just a few lines of code in a similar fashion.
```

File: en/style/styling-overview.md [hunk 6]
```diff
@@ -1,31 +1,36 @@
...
-With the Sketch libraries, you're able to modify the Indigo-Styling library to achieve the same results for both the Components and Patterns.
+The Sketch library has a few pages prefixed with a 🎨 emoji that provide assets and a powerful styling infrastructure upon which the rest of the library is built:
+
+- 🎨 Material Icons
+- 🎨 Colors
+- 🎨 Typography
+- 🎨 Elevation
+- 🎨 Illustrations
+
+In Sketch the colors and elevations exist as `Layer Styles`, while the typographies are defined as `Text Styles`, which allows to quickly change the theme for any given design we have created with the **Indigo.Design System**.
```
Comment (simeonoff) [inline en/style/styling-overview.md:19]:
```suggestion
In Sketch, the colors and elevations exist as `Layer Styles`, while typography is defined as `Text Styles`, which allows you to quickly change the theme for any given design we have created with the **Indigo.Design System**.
```

File: en/components/components-overview.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+---
+title: Components Overview - Design System Components
+_description: The Indigo Design System contains 60+ UI Components with numerous presets, states and elaborate styling capabilities built in.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+# Components Overview
+
+The **Indigo.Design System** empowers your design workflow with 60+ UI Components that also map to Ignite UI for Angular controls. Each component is optimized for responsive web development and provides various templating capabilities through smart layout and resizing rules set in the library. Furhtermore, we have added light/dark modes and various interaction states to choose from when designing dynamic experiences. All this streamlines application design and development allowing any user interface created with the library in Sketch to be built in Angular with Ignite UI.
```
Comment (YoannaIvanova) [inline en/components/components-overview.md:9]: Furhtermore -> Furthermore

File: en/style/styling-overview.md [hunk 8]
```diff
@@ -1,31 +1,36 @@
...
-## Indigo-Styling
-
-Indigo-Styling is the defacto representation of the theming engine. It is separated into four parts represented as different pages in the library: Icons, Colors, Typography and Elevations.
-
-<img class="responsive-img" src="../images/styling_structure.png"/>
+To find more:
 
-- [Material Icons](material-icons.md) - a collection of 100+ Material Icons, which you can easily extend
-- [Colors](colors.md) - a palette consisting of primary, secondary and special colors, as well as shades of gray and levels of opacity
+- [Material Icons](material-icons.md) - a collection of 150+ Material Icons extended with 250+ more icons created by us according to the Material icon design guidelines, which you can easily extend even furhter
```
Comment (YoannaIvanova) [inline en/style/styling-overview.md:29]: furhter -> further

**Review by radomirchev — APPROVED** [review]

**Pull request merged**